File: OdontoFlowAPI/OdontFlow/OdontFlow.Domain/ViewModel/OrderPayment/OrderPaymentViewModel.cs
namespace OdontFlow.Domain.ViewModel.OrderPayment;

public class OrderPaymentViewModel
{
    public Guid Id { get; set; }
    public string Barcode { get; set; }
    public string PatientName { get; set; }
    public string ClientName { get; set; }
    public PAYMENT_TYPE PaymentTypeId { get; set; }
    public string PaymentType { get; set; }
    public decimal Amount { get; set; }
    public string? Reference { get; set; }
    public DateTime CreationDate { get; set; }
}

File: OdontoFlowWeb/OdontFlow/src/app/core/model/order-payment-view.model.ts
export interface OrderPaymentViewModel {
    id: string;
    paymentTypeId: number;
    paymentType: string;
    amount: number;
    barCode? :string;
    patientName? :string;
    clientName? :string;
    reference?: string;
    creationDate?: Date;
  }

  export interface CreateOrderPaymentDto {
    orderId: string;
    paymentTypeId: number;
    amount: number;
    reference?: string;
  }

  export interface UpdateOrderPaymentDto  extends CreateOrderPaymentDto {
    id: string; 
  }

